rest/models/src/main/java/com/connexta/store/rest/models/QuarantineRequest.java

```diff
@@ -0,0 +1,98 @@
+/**
+ * Copyright (c) Connexta
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package com.connexta.store.rest.models;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.annotations.ApiModelProperty;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+/**
+ * Request
+ */
+public class QuarantineRequest {
+
+  @JsonProperty("metadataInfos")
+  @Valid
+  private List<MetadataInfo> metadataInfos = new ArrayList<>();
```

Comment:
✏️ The order of this doesn't matter, right? Can this be a `Set` instead?

Reply:
I am not sure how jackson will treat that. I am going to leave this as is for now since OpenAPI would generate it this way.